logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationRepository.kt

```diff
@@ -118,6 +119,8 @@ interface ConversationRepository {
         protocol: Conversation.Protocol,
         teamId: TeamId? = null
     ): Either<StorageFailure, List<QualifiedID>>
+
+    suspend fun fetchMlsOneToOneConversation(userId: UserId): Either<CoreFailure, Conversation>
```

Comment:
in some places we used OneOnOne and in some others OneToOne, I think we need to find a convention here and go with it!